fda-model/metrics/ConsequenceDistributionResults.cs

```diff
@@ -129,7 +145,15 @@ private void AddConsequenceRealizationByAssetCategory(double consequenceRealizat
             ConsequenceDistributionResult structureConsequenceDistributionResult = GetConsequenceResult(damageCategory, assetCategory, impactAreaID);
             if (structureConsequenceDistributionResult.IsNull)
             {
-                ConsequenceDistributionResult newStructureDamageResult = new ConsequenceDistributionResult(damageCategory, assetCategory, _ConvergenceCriteria, impactAreaID);
+                //HACK: hacking in bin width 
+                //TODO: Get rid of this terrible coding
+                double binWidth = 500;
```

Comment:
this needs to be fixed with priority

Reply:
This does seem distinctly not ideal. Do you have what you need to fix this before we merge?

Reply:
The fix will require a complete reorganization of how we're collecting damage so that we can be agnostic to units of measurement. This has been haunting me for the last two weeks, and I have not yet figured it out 100% but after a lot of trial and error, I think I see a path. Because it is a relatively complex refactor, I think I should do this work on a fresh branch.